docs/admin/authentication.md

```diff
@@ -486,6 +486,21 @@ An unsuccessful request would return:
 
 HTTP status codes can be used to supply additional error context.
 
+#### Using OpenStack Keystone authentication Webhook
+
+There are two parts to authentication with OpenStack Keystone. On the client-side, you can use `--auth-provider=openstack` option to reuse the `OS_*` environment variables:
```

Comment:
`s/authentication/authenticate`

Reply:
I suspect there's a word missing that would make this better than changing authentication to authenticate, which also reads oddly, perhaps better phrasing:
> There are two parts to support authentication with OpenStack Keystone.